app/2.0/start/first-element/intro.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Step 1: Set up"
+subtitle: "Build your first Polymer element"
+---
+
+<!-- toc -->
+
+In this tutorial, you’ll learn how to build elements using Polymer 2.0. You’ll
+create a simple Polymer element, a toggle button. The finished button will look
+something like this:
+
+![Sample star-shaped toggle buttons, showing pressed and unpressed
+state](/images/2.0/first-element/sample-toggles.png)
+
+You’ll be able to use it with simple markup like this:
+
+```html
+<icon-toggle></icon-toggle>
+```
+
+This project introduces you to most of the key concepts in working with
+Polymer.
+
+Don’t worry if you don’t understand everything. Each of the concepts presented
+here is described in detail in the Polymer documentation.
+
+
+## Step 1: Get set up
+
+To follow this tutorial, you’ll need:
+
+-   [git](https://git-scm.com/downloads).
+-   The starting code, [available on GitHub](https://github.com/PolymerLabs/polymer-2-first-element.git).
+-   The [Polymer CLI](/2.0/docs/tools/polymer-cli) to run the demo.
+
+### Fork the starting code repo
+
+1.  Fork the repo by running this command:
+
+    ```bash
+    git clone https://github.com/PolymerLabs/polymer-2-first-element.git
```

Comment:
We could save some time (like we did in 1.x) by supplying a downloadable file with the source code+dependencies pre-installed.